### PR TITLE
add examples of embedded dataset viewers

### PR DESCRIPTION
**Pull request by severo**

### CODE REVIEW

**Review by davanstrien — APPROVED** [review]
Very nice to see so much usage already :)

**Review by julien-c — COMMENTED** [review]
was not deployed for some reason (probably known?)

**Comment by severo** [comment]
I was waiting a bit before reporting again at https://huggingface.slack.com/archives/C023JAKTR2P/p1720796559706049... I'm not sure what is going on

File: docs/hub/datasets-viewer-embed.md [hunk 1]
```diff
@@ -55,3 +55,18 @@ It will open a modal with the iframe code that you can copy and paste into your
...
 <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/dataset-viewer-embed-search-button-modal-dark.png"/>
 </div>
+
+## Examples
+
+The embedded dataset viewer is used in multiple Machine Learning tools and platforms to display datasets. Here are a few examples:
+
+- [ZenML](https://www.zenml.io/blog/embedding-huggingface-datasets-visualizations-with-zenml)
+- [Metaflow + Outerbounds](https://www.linkedin.com/posts/eddie-mattia_the-team-at-hugging-facerecently-released-activity-7219416449084272641-swIu)
+- [AutoTrain](https://x.com/abhi1thakur/status/1813892464144798171)
```
Comment (julien-c) [inline docs/hub/datasets-viewer-embed.md:65]: for some of them you could add a screenshot or video/GIF of their UI

and mention hf username of creator / link to hf profile 👍  (potentially add a widget to follow them directly from here? or badge from https://huggingface.co/datasets/huggingface/badges – just an idea though)

Comment (severo) [reply docs/hub/datasets-viewer-embed.md:65]: yes, good ideas. Working on it

Comment (severo) [reply docs/hub/datasets-viewer-embed.md:65]: PR: https://github.com/huggingface/hub-docs/pull/1344

**Pull request merged**